internal/provider/types_streaming_namespace_policies.go

```diff
@@ -269,6 +269,11 @@ func getPulsarNamespacePolicies(ctx context.Context, pulsarAdminClient *pulsarad
 	policiesAttrTypes := plan.Policies.AttributeTypes(ctx)
 
 	resp, err := pulsarAdminClient.NamespacesGetPoliciesWithResponse(ctx, plan.Tenant.ValueString(), plan.Namespace.ValueString(), requestEditors...)
+	if resp.StatusCode() == 401 || resp.StatusCode() == 404 {
+		// we get back a 401 if the tenant is not present
```

Comment:
This seems like a bug/bad design in Pulsar if missing tenant is returning 401.  I guess there is not much we can do about it though.

Reply:
I can check this again, but I got a 401 the first time I tried, then a 404. Hopefully I just need a 404.

Reply:
Ok, so we get back a 401 if the tenant is not found, and a 404 if the namespace is not found. In either case, we need to remove the namespace from the TF state.